### ISSUE TITLE
TypeError: decoder.CorruptedError is not a function

### ISSUE THREAD

**Issue by jnovack** [Farof/heroprotocoljs#28]
https://github.com/Farof/heroprotocoljs/blob/master/lib/protocol32455.js#L400

I intentionally _(read: unintentionally, due to my poor coding)_ corrupted the data and found a bug.  This MAY be upstream as well.

The following line does NOT throw a `decoder.CorruptedError` as one would expect.

```
    if (typeid === null) throw new decoder.CorruptedError('eventid(' + eventid + ') at ' + decoder.toString());
```

**expected output**

```
Error: eventid(2) at buffer(6/4,[66]=71)
...stack trace...
```

**actual output**

```
/Users/jnovack/Source/heroprotocoljs/index.js:115
            for (let event of archive.protocol[decoderMap[archiveFile]](archive.readFile(archiveFile))) {
                                                                       ^

TypeError: decoder.CorruptedError is not a function
    at _decode_event_stream (/Users/jnovack/Source/heroprotocoljs/lib/protocol32524.js:401:32)
    at next (native)
    at Object.exports.decodeReplayGameEvents (/Users/jnovack/Source/heroprotocoljs/lib/protocol32524.js:425:21)
    at next (native)
    at Object.archive.getFile (/Users/jnovack/Source/heroprotocoljs/bin/storm.js:92:72)
    at Object.<anonymous> (/Users/jnovack/Source/heroprotocoljs/bin/storm.js:107:15)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
```

**Comment by Farof** [comment]
Thanks, corrected  75c7443.

**Comment by Farof** [comment]
Was super easy with automation BTW, corrected the template, executed the script, copied the files, commit, push. Didn't have to go thought each one by one.
